iron-form.html

```diff
@@ -343,7 +342,9 @@
     },
 
     _registerElement: function(e) {
-      var element = e.target;
+      // Get the actual element that fired the event
+      var element = Polymer.dom(e).path[0];
+
       element._parentForm = this;
       this._customElements.push(element);
```

Comment:
side question: why we have both `_parentForm` and `this._customElements` ? I think we can get rid of `_parentForm`